include/zephyr/sys/spsc_pbuf.h

```diff
@@ -31,11 +31,25 @@ extern "C" {
 #define SPSC_PBUF_UTILIZATION_OFFSET 8
 
 /**@} */
+#ifdef DCACHE
+#ifdef CONFIG_DCACHE_LINE_SIZE
```

Comment:
`CONFIG_DCACHE_LINE_SIZE` is actually an `int` symbol, so this should be `CONFIG_DCACHE_LINE_SIZE != 0` ?